src/v3/src/transformer/layout/idxTransformerMapping.ts

```diff
@@ -178,11 +178,18 @@ const TransformerMap: {
     },
   },
   [IDX_STEP.DEVICE_CHALLENGE_POLL]: {
+    [CHALLENGE_METHOD.APP_LINK]: {
+      transform: transformOktaVerifyDeviceChallengePoll,
+      buttonConfig: {
+        showDefaultSubmit: false,
+        showDefaultCancel: false,
+      },
+    },
     [CHALLENGE_METHOD.CUSTOM_URI]: {
       transform: transformOktaVerifyDeviceChallengePoll,
       buttonConfig: {
         showDefaultSubmit: false,
-        showDefaultCancel: true,
+        showDefaultCancel: false,
```

Comment:
Ah, did I get this one wrong? I guess we actually need to add the cancel on our own in a custom way.

Reply:
Yeah looks like a custom cancel link was needed for all methods for cancel polling